common/internal/unclosedresponsebodymonitor_test.go

```diff
@@ -29,18 +29,15 @@ func TestCheckForUnclosedResponses(t *testing.T) {
 
 	// test
 	require.Panics(t, func() {
-		CheckForUnclosedResponses(ctx, logger.WithField("test", "test"))
+		CheckForUnclosedResponses(ctx)
 	})
-
-	for _, entry := range hook.Entries {
```

Comment:
Can we add a simple check here? It doesn't need to be equivalent but at least we should verify that something is logged.